image-layout.md

```diff
@@ -115,4 +116,42 @@ $ cat ./blobs/sha256/e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7f
 [tar stream]
 ```
 
+## Index 
+
+Index is intended to be generated from the content in an image-layout after any updates have been made.
+Allows for mapping random access to the references in the image-layout, and needing only to read from the `oci-layout` once to build this map of references.
```

Comment:
As it stands, the line is not a sentence (that would be “The index allows…” or some such).  And you already have random access with `refs/`.  The think the index adds is single-file list support when you can't walk `refs/` with `readdir(3)` or similar.  And the HTTP sentence makes that clear with a concrete example.